clarity_ext/dilution.py

```diff
@@ -225,6 +228,18 @@ def __init__(self, artifact_service, robot_name, scale_up_low_volumes=True,
         self.split_up_high_volume_rows()
         self.do_positioning()
         self.sort_transfers()
+        self.grouped_transfers = list(self._grouped_transfers())
+
+    def _grouped_transfers(self):
+        """
+        Sometimes a sample transfer from source A to destination B is
+        split up on several rows, due to the max pipetting volume of 50 ul.
+        :return: An iterable group of transfers for a common destination well.
+        """
+        for key, transfer_group in groupby(
```

Comment:
One question: I assume that the data has already been sorted using the same key (since groupby generally requires it to be). Might break in marginal cases otherwise.

https://docs.python.org/2/library/itertools.html#itertools.groupby